<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_thlete_from_team_page</class>
 <widget class="QDialog" name="delete_thlete_from_team_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>135</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="team_name_input"/>
    </item>
    <item>
     <widget class="QLineEdit" name="athlete_name_input"/>
    </item>
    <item>
     <widget class="QPushButton" name="remove_button">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="team_name_lable">
      <property name="text">
       <string>Team Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="athlete_name_lable">
      <property name="text">
       <string>Athlete Name:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="team_mode_input">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Football</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PingPong</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Badminton</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
